docs/source/CartesianConfig.rst

```diff
@@ -87,10 +93,21 @@ combinations and names::
     dict    8:  six.two
     dict    9:  six.three
 
-Variant shortnames represent the <TESTNAME> value used when results are
-recorded (see section Job Names and Tags. For convenience
-variants who’s name begins with a ‘``@``’ do not prepend their name to
-'short-name', only 'name'. This allows creating ‘shortcuts’ for
+Dict 1 elements::
+
+    key1 = Hello
+    key3 = foo
+
+Dict 7 elements::
+
+    key1 = foo
+    key2 = bar
+
+These combinations names can become quite long and, due to this, a 'short name'
+is also used. A combination short name represent the <TESTNAME> value used
+when a combination is recorded (see section Job Names and Tags. For convenience,
+variants which name begins with a ‘``@``’ do not prepend their name to
+'short name', only 'name'. This allows creating ‘shortcuts’ for
```

Comment:
Yep, I like this better than the original. As for the style it could be improved by using `` but it is readable and better so I don't think we have to be picky...